apps/language_server/lib/language_server/providers/code_lens/test.ex

```diff
@@ -70,17 +71,18 @@ defmodule ElixirLS.LanguageServer.Providers.CodeLens.Test do
       CodeLens.build_code_lens(block.line, "Run tests", @run_test_command, %{
         "filePath" => file_path,
         "describe" => block.name,
-        "projectDir" => project_dir
+        "projectDir" => project_dir,
+        "module" => Atom.to_string(block.module) |> String.replace("Elixir.", "")
```

Comment:
nit: commit to the pipeline:
```suggestion
        "module" => block.module |> Atom.to_string() |> String.replace("Elixir.", "")
```

Reply:
alternately:
```elixir
"module" => block.module |> Module.split() |> Enum.join(".")
```

Reply:
This code has repeated three times, it's likely time to make this a function.